include/GafferUI/AnimationGadget.h

```diff
@@ -74,6 +74,9 @@ class GAFFERUI_API AnimationGadget : public Gadget
 		void setContext( Gaffer::Context *context );
 		Gaffer::Context *getContext() const;
 
+		bool onTimeAxis( const Imath::V2i& rasterPosition ) const;
+		bool onValueAxis( const Imath::V2i& rasterPosition ) const;
```

Comment:
Equivalent methods on other gadget types take `const IECore::LineSegment3f &line` instead of `const Imath::V2i& rasterPosition`, so that all operations on gadgets take place in their local space (see SceneGadget::objectAt, ImageGadget::pixelAt, ConnectionGadget::endAt etc). This is convenient for client code, because `Gadget::mouseMoveSignal()` etc give you the exact `event.line` you need directly. We should follow that convention here too.

Reply:
The envisaged use for these functions is to be called from a context menu signal which does not have an event line so would be called with ````onTimeAxis( GafferUI.Widget.mousePosition( relativeTo = gadget ) )```. On the other hand if they take an event line (as you have requested) they would need to be called with ```position = GafferUI.Widget.mousePosition( relativeTo = gadget ); line = gadget.getViewportGadget().rasterToGadgetSpace( imath.V2f( position.x, position.y ), gadget = gadget ); onTimeAxis( line )``` which seems much less convienient